<launch>

    <!-- Start follow_vehicle nodes -->

    <node name="gimbal_control" pkg="summer_school_exercises" type="gimbal_control.py" output="screen" ns="drone_1">
        <param name="drone_id" value="1"/>
    </node>

</launch>
